plyer/platforms/win/libs/wifi_defs.py

```diff
@@ -5,8 +5,24 @@
 /ms705945%28v=vs.85%29.aspx
 '''
 
-from ctypes import *
-from ctypes.wintypes import *
+from ctypes import (
```

Comment:
Are we sure these are all the used imports?

Reply:
Sure. In addition, linter is not complaining about undefined names.